packages/dynamic_layouts/example/lib/main.dart

```diff
@@ -0,0 +1,115 @@
+import 'package:flutter/material.dart';
```

Comment:
Just double checking my own sanity (fever 😅 ), this is not the sample app, it's the newly created app while debugging the CI issue?